docker/docker-python/Dockerfile.mxnet.python.cpu

```diff
@@ -19,10 +19,11 @@
 # Dockerfile to build MXNet for CPU
 
 FROM ubuntu:16.04
+ARG version
 
-RUN apt-get update
-RUN apt-get install -y wget python gcc
-RUN wget https://bootstrap.pypa.io/get-pip.py
-RUN python get-pip.py
+RUN apt-get update && \
+    apt-get install -y wget python gcc && \
+    wget https://bootstrap.pypa.io/get-pip.py && \
+    python get-pip.py
 
-RUN pip install mxnet
+RUN pip install mxnet==$version
```

Comment:
I think we should separate the mxnet package installation from the dependency installation and rather do it during runtime. Otherwise, the package will be cached by our Docker cache and we won't be running the live version.

Reply:
This can be avoided by adding the `--no-cache` flag to the docker build command too, right?

Reply:
Right, but we want to use the cache. We just have to separate runtime dependencies from build time dependencies.